modules/lvm2/udiskslvm2moduleiface.c

```diff
@@ -154,7 +154,6 @@ lvm_update_from_variant (GPid      pid,
           udisks_linux_volume_group_object_destroy (group);
           g_dbus_object_manager_server_unexport (manager,
                                                  g_dbus_object_get_object_path (G_DBUS_OBJECT (group)));
-          g_object_unref (G_OBJECT (group));
```

Comment:
@vpodzime The original design appears to disregard any protection to concurrent access to this hash table. From looking at the code and considering what I am seeing with #110 I'm starting to think that having concurrent access to the same hash table is indeed possible, which would be bad as the GHashTable is not thread safe.  I was planning on looking at #110 some more, so when I am looking at that I will see if I can convince myself that this issue does or does not exist (concurrent access to same hash table).